### ISSUE TITLE
Need to allow '/' character in field names to support customer namespaces

### ISSUE THREAD

**Issue by ryanwise** [sbcgua/abap_data_parser#8]
This line checks the incoming column names for invalid characters:

https://github.com/sbcgua/abap_data_parser/blob/0c1f49fc379ac24062ae596962880ce576f28cf0/src/zcl_text2tab_parser.clas.abap#L438

however, it disqualifies my valid column names which begin with a namespace, such as '/CUST/COLUMN'.  Can we add the character to the whitelist?

thanks,
ryan

**Comment by sbcgua** [comment]
Hmm, I actually took this string from some internal validation routine. Are you absolutely sure it is a valid field name?
If yes, PR is welcomed :)

**Comment by ryanwise** [comment]
In my sandbox, once I add the character to the list, my data loads correctly (using mockup loader to simulate data).

I noticed the comment regarding CL_ABAP_STRUCTDESCR->CHECK_COMPONENT_TABLE and I consulted the source there.  It looks like it validates the presence and validity of the namespace first, saves the character offset, and then proceeds with the validation string used above.  I'll attempt to capture than in a PR and hopefully not break anything!
